README.md

```diff
@@ -24,47 +24,105 @@ TODO
  - `npm install --save k-simple-state`
 
 ## API
+### createStore(<store_description>, \<options>)
+| parameter | required | description |
+|---|---|---|
+| <store_description>| required | object describing your store, it can be nested |
+| [\<options>](#options) | optional | all options you may want to override |
+
+#### options
+All options are optionals to keep the default usage as simple as possible.
+
+| key | type | default | description |
+|---|---|---|---|
+| listeners | `array` | `undefined` | array of all listeners. Listeners are a big part of this lib, you can [click here for detail](#TODO). |
+| devtools | `boolean` | `true` | [redux-devtools](https://github.com/zalmoxisus/redux-devtools-extension) is activated. |
+| hideRedux | `boolean` | `true` | actions and selectors from [`k-redux-factory`](https://github.com/alakarteio/k-redux-factory) are used without specifying `dispatch` or `getState`. |
+| enhancer | redux enhancer | `undefined` | usual `compose` and `applyMiddlare` you already use with Redux can be injected here (like router, redux-saga, etc). <br />`compose` and `applyMiddleware` from Redux are exposed by the lib. To use them:<br /> ```import { compose, applyMiddleware } from 'k-simple-state'```. |
+| init | `object` | `undefined` | the default value of your store. |
+
+#### listeners
 TODO
 
 ## Examples
-Full example
-```es6
-const { createState } = require('k-simple-state')
+### Simple one, with k-redux-factory
+```js
+import { createStore } from 'k-simple-state'
 
-const saveToLocalStorage = store => next => (action) => {
-  // dispatch action
-  const res = next(action)
+// create a store of todos
+const store = createStore({
+  todos: { type: 'keyValue', key: 'id' },
+})
 
-  // save store to localStorage
-  window.localStorage.setItem('redux-store', JSON.stringify(store.getState()))
+// this next line will dispatch a k-redux-factory action and update the store
```

Comment:
`// dispatch a k-redux-factory action and update the store`

Reply:
And if I don't have to speak about k-redux-factory ? :)

`// dispatch a k-simple-state action and update the store` ?